src/index.js

```diff
@@ -19,7 +21,7 @@ module.exports.unpack = function({inputPath, outputPath}) {
     }
 
     const source = fs.createReadStream(input);
-    const target = tar.extract('/tmp');
+    const target = tar.extract(outputBaseDir);
```

Comment:
I didn't like the idea of rimraf'ing `/tmp/*` for test runs so I made the target dir configurable.

Reply:
In hindsight it didn't matter because it eventually ran on a 1-shot docker container. Let me know if you want this bit reverted.